caddytls/certificates.go

```diff
@@ -178,7 +179,19 @@ func makeCertificate(certPEMBlock, keyPEMBlock []byte) (Certificate, error) {
 	if err != nil {
 		log.Printf("[WARNING] Stapling OCSP: %v", err)
 	}
-
+	if certificateTransparency {
+		logs, err := getTrustedCTLogs()
```

Comment:
It would be nice if we could get this at startup and then cache it for some time, so that each time we load a certificate (which might even be during TLS handshakes), we don't have to do a GET request and rely on that succeeding each time.